versions.props

```diff
@@ -48,3 +48,4 @@ com.esotericsoftware:kryo = 4.0.2
 org.eclipse.jetty:* = 9.4.43.v20210629
 org.testcontainers:* = 1.17.5
 io.delta:delta-core_* = 2.2.0
+com.github.luben:zstd-jni = 1.5.4-1
```

Comment:
Why is this new dependency required?

Reply:
It is needed because new tests added run with a few different codecs, including zstd. Without the dependency, we get
```
    java.lang.NoClassDefFoundError: com/github/luben/zstd/ZstdOutputStreamNoFinalizer
        at org.apache.avro.file.ZstandardCodec.compress(ZstandardCodec.java:73)
```

Reply:
However, this dependency is only needed at test runtime, so I'll change the dependency from `testImplementation` to `testRuntimeOnly`.